code/modules/mining/equipment/wormhole_jaunter.dm

```diff
@@ -56,27 +64,24 @@
 	if(. & EMP_PROTECT_SELF)
 		return
 
-	var/mob/M = loc
-	if(istype(M))
-		var/triggered = FALSE
-		if(M.get_item_by_slot(ITEM_SLOT_BELT) == src)
-			if(power == 1)
-				triggered = TRUE
-			else if(power == 2 && prob(50))
-				triggered = TRUE
```

Comment:
Where did this logic go?

Reply:
The EMP code checked if the jaunter was in the beltslot of a mob before an EMP could affect it.  My changes removed that requirement.